docs/DeveloperGuide.adoc

```diff
@@ -96,22 +96,22 @@ When you are ready to start coding,
 [[Design-Architecture]]
 === Architecture
 
-.Architecture Diagram
-image::Architecture.png[width="600"]
+The *_Architecture Diagram_* (Refer to Figure 1) explains the high-level design of the App. Given below is a quick overview of each component.
 
-The *_Architecture Diagram_* given above explains the high-level design of the App. Given below is a quick overview of each component.
+image::Architecture.png[width="600"]
+_Figure 1. Architecture Diagram_
 
 [TIP]
-The `.pptx` files used to create diagrams in this document can be found in the link:{repoURL}/docs/diagrams/[diagrams] folder. To update a diagram, modify the diagram in the pptx file, select the objects of the diagram, and choose `Save as picture`.
+The `.pptx` files used to create diagrams in this document can be found in the link:{repoURL}/docs/diagrams/[diagrams] folder. To update a diagram, modify the diagram in the .pptx file, select the objects of the diagram, and choose `Save as picture`.
```

Comment:
Can this tip be placed elsewhere? Or do you think this is the most appropriate location?

Reply:
Perhaps it could be placed elsewhere. Where do you think it should be placed?

Reply:
How about if we place it right after the table of contents?